### PR TITLE
T&D frequency: stream start, 6h(*)

### LINKED ISSUE #30175: Adjust T&D times to be once every 6h

### PR DESCRIPTION
**Pull request by evantahler**
closes: https://github.com/airbytehq/airbyte/issues/30175
re: https://github.com/airbytehq/airbyte-internal-issues/issues/2585

We still want to T&D early, but then less often over longer syncs

### CODE REVIEW

File: airbyte-integrations/bases/base-typing-deduping/src/test/java/io/airbyte/integrations/base/destination/typing_deduping/TypeAndDedupeOperationValveTest.java [hunk 1]
```diff
@@ -54,47 +54,22 @@ public void testReadyToTypeAndDedupe() {
...
     // method call increments time
     Assertions.assertFalse(valve.readyToTypeAndDedupe(STREAM_A));
     // More than enough time has passed now
-    elapseTime(minuteUpdates, 15);
+    elapseTime(minuteUpdates, 1000 * 60 * 60 * 2);
     Assertions.assertTrue(valve.readyToTypeAndDedupe(STREAM_A));
   }
 
-  @Test
-  public void testIncrementInterval() {
```
Comment (evantahler) [inline airbyte-integrations/bases/base-typing-deduping/src/test/java/io/airbyte/integrations/base/destination/typing_deduping/TypeAndDedupeOperationValveTest.java:67]: I really didn't understand this test... so I removed it

File: airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/TypeAndDedupeOperationValve.java [hunk 2]
```diff
@@ -16,9 +16,6 @@
 public class TypeAndDedupeOperationValve extends ConcurrentHashMap<AirbyteStreamNameNamespacePair, Long> {
 
   private static final long NEGATIVE_MILLIS = -1;
-  private static final long FIFTEEN_MINUTES_MILLIS = 1000 * 60 * 15;
-  private static final long ONE_HOUR_MILLIS = 1000 * 60 * 60 * 1;
-  private static final long TWO_HOURS_MILLIS = 1000 * 60 * 60 * 2;
   private static final long FOUR_HOURS_MILLIS = 1000 * 60 * 60 * 4;
```
Comment (edgao) [inline airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/TypeAndDedupeOperationValve.java:19]: nit: 6 hours? (though I don't think it really matters tbh)

**Review by edgao — APPROVED** [review]
lgtm, one nonblocking nit on the timing numbers

File: airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/TypeAndDedupeOperationValve.java (hunk 2 above)
Comment (evantahler) [reply airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/TypeAndDedupeOperationValve.java:19]: :doh-homer-emoji:

**Pull request merged**